app/models/user.ts

```diff
@@ -8,7 +8,7 @@ export class User {
   @Column('varchar')
   name: string;
 
-  @Column('varchar', { name: 'last_name' })
+  @Column('varchar')
   lastName: string;
```

Comment:
why delete `name` tag?
remember to save in db, according to [wolox doc](https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#33--database), we should use snake_case format.